### DISCUSSION TITLE
Is there a demo that shows how to configure different color schemes to the leaflet map?

### DISCUSSION THREAD

**Discussion by abishekrsrikaanth** [Leaflet/Leaflet#8278] [Q&A]
Is there a demo that shows how to configure different color schemes to the leaflet map? I am trying to customize the leaflet map based on the current theme of the page and would like to know how to do that?

**Comment by mourner** [comment]
It depends on which map tiles provider you are using with Leaflet, but in general many commercial services provide tools for customizing the map style. The most advanced of such tools which I can recommend is Mapbox Studio https://www.mapbox.com/mapbox-studio/

**Comment by Malvoz** [comment]
If this is about styling the UI to match the theme of the web page (how I initially interpreted it) then CSS variables should be very helpful if utilized in leaflet.css in the future: https://github.com/Leaflet/Leaflet/discussions/7729#discussioncomment-2601223.
